tests/support/rally.test.mjs

```diff
@@ -28,6 +28,7 @@ describe('Rally', function () {
         "key-id",
         {},
         true, // Developer mode.
+        () => {},
```

Comment:
Can we also add a new test here to make sure that we throw if no callback is provided?

Reply:
I meant to bring this up, thanks... since we're using positional arguments (as opposed to an options object or parameter desctructuring etc) then it seems like we'd need to enforce all of the options (throw if any argument is not provided or an incorrect type)